### PR TITLE
Add types.json file

### PR DESCRIPTION
**Pull request by shelbyd**

### CODE REVIEW

**Review by juliosantos — COMMENTED** [review]
I don't know enough to approve this, but from your previous explanation of what this meant, I take these are chain metadata so that Subscan can know how to interface with it (I would have expected this to be also needed to make polkadot.js apps works, is it different there?)

I take this to mean "hey, so, if you see "FractalId", don't get confused, it's a u64. So when rendering, render is as such. Am I off base?

**Comment by shelbyd** [comment]
These types are used in all sorts of things related to Substrate. Because of that, we should have the canonical one in source.

This isn't only for Subscan, it's needed in Polkadot JS, and anything using the polkadot api. It is for rendering and understanding custom types.

**Review by juliosantos — APPROVED** [review]

File: scripts/download_chain.sh [hunk 1]
```diff
@@ -4,4 +4,4 @@ set -euxo pipefail
 
 sub2sql --chain=wss://nodes.testnet.fractalprotocol.com \
   --out=testnet.sqlite \
-  --types='{"FractalId": "u64", "MerkleTree": "Raw"}'
+  --types="$(cat blockchain/types.json | tr -d '\n')"
```
[the remark this answers is not in the document]
Comment (shelbyd) [reply scripts/download_chain.sh:7]: It's a dependency for development, certainly not for production. I have on my TODO list to add Readme data before mainnet, but it's not super high priority.

[the remark this answers is not in the document]
Comment (shelbyd) [reply scripts/download_chain.sh:7]: I'm happy to distribute sub2sql on whatever channels you want. I have documentation on the page to explain how to use it.

**Pull request merged**